api/src/main/java/io/strimzi/api/kafka/model/bridge/KafkaBridgeHttpConfig.java

```diff
@@ -38,8 +38,14 @@ public class KafkaBridgeHttpConfig implements UnknownPropertyPreserving, Seriali
 
     public static final int HTTP_DEFAULT_PORT = 8080;
     public static final String HTTP_DEFAULT_HOST = "0.0.0.0";
+    public static final long HTTP_DEFAULT_TIMEOUT = -1L;
+
+    private String host = HTTP_DEFAULT_HOST;
     private int port = HTTP_DEFAULT_PORT;
     private KafkaBridgeHttpCors cors;
+    private long timeoutSeconds = HTTP_DEFAULT_TIMEOUT;
+    private KafkaBridgeProducerEnablement producer = new KafkaBridgeProducerEnablement();
+    private KafkaBridgeConsumerEnablement consumer = new KafkaBridgeConsumerEnablement();
```

Comment:
I think this needs a bit more discussion and thinking ... if we create this as a new object, then it will be because we expect we might need additional fields there. So it maybe should be called `KafkaBridgeProducerConfig` / `KafkaBridgeConsumerConfig`? But wouldn't it make sense to somehow try to incorporate it into `the .spec.consumer` / `.spec.producer` sections?

Reply:
That's the point I was going to make as well.
My view, is that what's under `http` is related to HTTP. So when I refer to something like `http.producer.enabled` and `http.consumer.enabled` I am referring to the HTTP part of the corresponding Kafka producer and consumer. The current status is that I can just enable/disable the usage of consumer or producer on the HTTP side, which means being or not being able to use HTTP to producer and/or consume (so by using the corresponding Kafka part).
Of course, enabling/disabling the HTTP part of it would mean enabling/disabling the corresponding Kafka part which comes to the @scholzj suggestion to use something like `.spec.consumer.enabled` and `.spec.producer.enabled`. That part was mostly there for Kafka related configuration but doesn't mean we can't change it. I agree it seems too much having a dedicated class unless we envisage additional HTTP producer/consumer related configuration which I can't think about now.
Anyway, I would say it needs further discussion in terms of API. I think we also said it would need a proposal first where we can make a decision all together. https://github.com/strimzi/strimzi-kafka-operator/issues/8732#issuecomment-1612639586

@steffen-karlsson would you mind to open a PR with a proposal here https://github.com/strimzi/proposals which could mimic what you have here and starting the discussion on it?

Reply:
Sure, I raised this to @ppatierno in https://github.com/strimzi/strimzi-kafka-operator/issues/8732 for discussion, if it should be separate object or inside the `spec.consumer` and `spec.producer`

Reply:
It seems we are having the same discussion on different places :-D
I think a new proposal will be the only definitive one.

Reply:
Sure will fix the other comment from @scholzj and then raise a proposal for further discussion :)

Reply:
Hello @scholzj and @ppatierno 

I've raised a proposal with the two implementation options here: https://github.com/strimzi/proposals/pull/111